### PR TITLE
[Feature] VIP Integration

### PR DESCRIPTION
**Pull request by JasonMa2016**
## Description

1. VIP model code in `torchrl/transforms/vip.py`.
2. Testing code appended to `test/test_transforms.py`.
3. Ensured that all tests pass.

## Motivation and Context

New pre-trained visual encoder for RL from FAIR. 

## Types of changes

What types of changes does your code introduce? Remove all that do not apply:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds core functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation (update in the documentation)
- [ ] Example (update in the folder of examples)

## Checklist

Go over all the following points, and put an `x` in all the boxes that apply.
If you are unsure about any of these, don't hesitate to ask. We are here to help!

- [x] I have read the [CONTRIBUTION](https://github.com/facebookresearch/rl/blob/main/CONTRIBUTING.md) guide (**required**)
- [ ] My change requires a change to the documentation.
- [x] I have updated the tests accordingly (*required for a bug fix or a new feature*).
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by vmoens** [comment]
Thanks for this @JasonMa2016, wonderful contribution!
Do we want to design a reward transform while we're at it? Or should this be part of a separate PR?

**Comment by JasonMa2016** [comment]
Let's make reward transform a separate PR! We can discuss what's the best way to go about this.
